server/apps/trak/models/site_permission.py

```diff
@@ -50,3 +51,11 @@ class SitePermission(models.Model):
 
     def __str__(self):
         return f'{self.profile.user}: {self.site}'
+
+    def clean(self):
+        if self.site_manager:
+            fields = ['annual_report', 'biennial_report', 'e_manifest', 'my_rcra_id', 'wiets']
+            for field_name in fields:
+                field_value = getattr(self, field_name)
+                if field_value != "Certifier":
+                    raise ValidationError(f"The value for the '{field_name}' field must be set to 'Certifier'.")
```

Comment:
@nihalrahmanqb This looks nice and concise. I think we can shorten in further if we inline the if statement so it reads like the following.

```python
if getattr(self, field_name) != "Certifier":
    raise ValidationError(f"The value for the '{field_name}' field must be set to 'Certifier'.")
```

Reply:
Hello @dpgraham4401, thank you for your suggestion. I have made the necessary changes to the pull request. I will work on writing the regression tests for the code and submit another pull request once they are done I think it will take some time for me to do that as I have to learn a little more.